requirements.txt

```diff
@@ -2,66 +2,69 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile --resolver=backtracking ./requirements.in
+#    pip-compile ./requirements.in
 #
-anyio==3.6.2
+anyio==3.7.1
     # via httpcore
-attrs==22.1.0
+attrs==23.1.0
     # via jsonschema
-certifi==2022.12.7
+certifi==2023.7.22
     # via
     #   httpcore
     #   httpx
     #   requests
-chardet==3.0.4
+charset-normalizer==3.2.0
     # via requests
+exceptiongroup==1.1.2
+    # via anyio
 h11==0.14.0
     # via httpcore
-httpcore==0.16.2
+httpcore==0.17.3
     # via httpx
-httpx==0.23.1
+httpx==0.24.1
     # via lightkube
-idna==2.10
+idna==3.4
     # via
     #   anyio
+    #   httpx
     #   requests
-    #   rfc3986
+importlib-resources==6.0.0
+    # via jsonschema
 jinja2==3.1.2
     # via -r ./requirements.in
-jsonschema==3.2.0
+jsonschema==4.17.3
     # via serialized-data-interface
-lightkube==0.11.0
+lightkube==0.14.0
     # via -r ./requirements.in
-lightkube-models==1.25.4.4
+lightkube-models==1.27.1.4
     # via lightkube
-markupsafe==2.1.1
+markupsafe==2.1.3
     # via jinja2
-ops==1.5.4
+ops==2.4.1
```

Comment:
did you try pinning ops<2 to get at least 1.5.5 and not introduce such a big change?

Reply:
Just tried, and it brings SDI 0.7.0. 
```
+ops==1.5.5
     # via
     #   -r ./requirements.in
     #   serialized-data-interface
. . .
-serialized-data-interface==0.3.6
+serialized-data-interface==0.7.0
```
I am not sure how it is possible, since in .in file we have:
```
# dex-auth's current implementation is not compatible
# with SDI>0.3.
serialized-data-interface<4.0

```

Reply:
Done.